lib/ansible/modules/command.py

```diff
@@ -336,14 +344,34 @@ def main():
                 rc=0
             )
 
+    if changes:
+        # do not run the command if the line contains changes=filename
+        # and the filename does not exist.  This allows idempotence
+        # of command executions.
```

Comment:
This comment can be more succint.
```suggestion
        # Do not run the command and exit now if the file does not exist
```